src/test/scala/de/zalando/beard/filter/FilterSpec.scala

```diff
@@ -0,0 +1,36 @@
+package de.zalando.beard.filter
+
+import org.scalatest.{FunSpec, Matchers}
+
+/**
+  * @author boopathi
+  */
+class FilterSpec extends FunSpec with Matchers {
+  def getFilter(name: String) =
```

Comment:
this method should be private and usually you put the private methods at the end of the file...